src/main/java/firsttask/taskmanager/Controller/Login.java

```diff
@@ -0,0 +1,49 @@
+package firsttask.taskmanager.Controller;
+
+import firsttask.taskmanager.Models.AuthenticationResponse;
+import firsttask.taskmanager.Models.AuthenticationRequest;
+import firsttask.taskmanager.Security.JWTSecurity.JwtUtil;
+import firsttask.taskmanager.Security.UserDetailsServiceImpl;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.BadCredentialsException;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class Login {
+
+    @Autowired
+    private AuthenticationManager authenticationManager;
+    @Autowired
+    private UserDetailsServiceImpl userDetailsService;
+    @Autowired
+    private JwtUtil jwtTokenUtil;
+    private static final Logger logger = LoggerFactory.getLogger(UserController.class);
```

Comment:
We usually use capital letters for static final variables
LOGGER instead of logger